clang/test/dpct/thrust-algo.cu

```diff
@@ -13,10 +13,14 @@
 #include <thrust/extrema.h>
 #include <thrust/gather.h>
```

Comment:
can the migrated code can be build by icx?

Reply:
Yes，https://github.com/oneapi-src/SYCLomatic-test/pull/178 has been submitted to verify the build&run status.